### PR TITLE
fix NPE in KubernetesSecretApplier when looping through component's aliases

### PR DESCRIPTION
**Pull request by sparkoo**
### What does this PR do?
Fix NPE caused by looping through component aliases, that are not guaranteed to be non-null.

### What issues does this PR fix or reference?
n/a

### CODE REVIEW

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public abstract void applySecret(E env, RuntimeIdentity runtimeIdentity, Secret
         return env.getDevfile()
             .getComponents()
             .stream()
-            .filter(c -> c.getAlias().equals(componentName))
+            .filter(c -> componentName.equals(c.getAlias()))
```
Comment (metlos) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java:62]: Is the component name guaranteed to be non-null?

**Review by metlos — COMMENTED** [review]
LGTM, iff this is not swapping one NPE for another :)

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java (hunk 1 above)
Comment (sparkoo) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java:62]: line 58:      if (componentName != null) {

**Review by skabashnyuk — APPROVED** [review]

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java (hunk 1 above)
Comment (metlos) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/provision/secret/KubernetesSecretApplier.java:62]: :scream: I'm blind, I'm blind!

**Review by metlos — APPROVED** [review]

**Pull request merged**